controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -433,20 +430,22 @@ func (r *StorageConsumerReconciler) reconcileCephClientRBDNode() error {
 		return nil
 	})
 
-	cephClientRBDNodeDetails := &ocsv1alpha1.CephResourcesSpec{
-		Kind: r.cephClientRBDNode.Kind,
-		Name: r.cephClientRBDNode.Name,
+	if err != nil {
+		r.Log.Error(
+			err,
+			"Failed to update CephClient.",
+			"CephClient",
+			klog.KRef(r.cephClientRBDNode.Namespace, r.cephClientRBDNode.Name),
+		)
+		return err
 	}
+
+	var phase string
```

Comment:
To make it more obvious that we are willing to intentionally use an empty string
```suggestion
phase := "" 
```